src/locale/locales/pl/messages.po

```diff
@@ -0,0 +1,8038 @@
+
+#: src/view/com/modals/InviteCodes.tsx:183
+msgid "Copied"
+msgstr "Skopiowane"
+
+#: src/screens/Settings/AboutSettings.tsx:66
+msgid "Copied build version to clipboard"
+msgstr "Skopiowano wersję"
+
+#: src/components/dms/MessageMenu.tsx:57
+#: src/lib/sharing.ts:25
+#: src/view/com/modals/InviteCodes.tsx:153
+#: src/view/com/util/forms/PostDropdownBtn.tsx:240
+#: src/view/com/util/post-ctrls/PostCtrls.tsx:386
+msgid "Copied to clipboard"
+msgstr "Skopiowano"
+
+#: src/components/dialogs/Embed.tsx:136
+#: src/screens/Settings/components/CopyButton.tsx:66
+msgid "Copied!"
+msgstr "Skopiowano!"
+
+#: src/components/StarterPack/QrCodeDialog.tsx:175
+msgid "Copy"
+msgstr "Kopiuj"
+
+#: src/screens/Settings/components/AddAppPasswordDialog.tsx:196
+msgid "Copy App Password"
+msgstr "Kopiuj hasło aplikacji"
+
+#: src/screens/Settings/AboutSettings.tsx:61
+msgid "Copy build version to clipboard"
+msgstr "Kopiuj wersję"
+
+#: src/components/dialogs/Embed.tsx:122
+#: src/components/dialogs/Embed.tsx:141
+msgid "Copy code"
+msgstr "Kopiuj kod"
+
+#: src/screens/Settings/components/ChangeHandleDialog.tsx:472
+msgid "Copy DID"
+msgstr "Kopiuj DID"
+
+#: src/screens/Settings/components/ChangeHandleDialog.tsx:405
+msgid "Copy host"
+msgstr "Kopiuj host"
+
+#: src/components/StarterPack/ShareDialog.tsx:124
+msgid "Copy link"
+msgstr "Kopiuj link"
+
+#: src/components/StarterPack/ShareDialog.tsx:131
+msgid "Copy Link"
+msgstr "Kopiuj link"
+
+#: src/view/screens/ProfileList.tsx:487
+msgid "Copy link to list"
+msgstr "Kopiuj link do listy"
+
+#: src/view/com/util/forms/PostDropdownBtn.tsx:452
+#: src/view/com/util/forms/PostDropdownBtn.tsx:461
+msgid "Copy link to post"
+msgstr "Kopiuj link do wpisu"
+
+#: src/components/dms/MessageMenu.tsx:110
+#: src/components/dms/MessageMenu.tsx:112
+msgid "Copy message text"
+msgstr "Kopiuj tekst wiadomości"
+
+#: src/view/com/util/forms/PostDropdownBtn.tsx:430
+#: src/view/com/util/forms/PostDropdownBtn.tsx:432
+msgid "Copy post text"
+msgstr "Kopiuj tekst wpisu"
+
+#: src/components/StarterPack/QrCodeDialog.tsx:169
+msgid "Copy QR code"
+msgstr "Kopiuj kod QR"
+
+#: src/screens/Settings/components/ChangeHandleDialog.tsx:426
+msgid "Copy TXT record value"
+msgstr "Kopiuj rekord TXT"
+
+#: src/Navigation.tsx:284
+#: src/view/screens/CopyrightPolicy.tsx:31
+msgid "Copyright Policy"
+msgstr "Polityka praw autorskich"
+
+#: src/components/dms/LeaveConvoPrompt.tsx:39
+msgid "Could not leave chat"
+msgstr "Awaria podczas opuszczania rozmowy"
+
+#: src/view/screens/ProfileFeed.tsx:104
+msgid "Could not load feed"
+msgstr "Awaria podczas ładowania kanału"
+
+#: src/view/screens/ProfileList.tsx:1020
+msgid "Could not load list"
+msgstr "Awaria podczas ładowania listy"
+
+#: src/components/dms/ConvoMenu.tsx:88
+msgid "Could not mute chat"
+msgstr "Awaria podczas wyciszania rozmowy"
+
+#: src/view/com/composer/videos/VideoPreview.web.tsx:56
+msgid "Could not process your video"
+msgstr "Awaria podczas przetwarzania wideo"
```

Comment:
`Awaria` is kind of catastrophic :) like server malfunction.
`Błąd` sounds more user friendly.

```suggestion
msgid "Could not leave chat"
msgstr "Błąd podczas opuszczania rozmowy"

#: src/view/screens/ProfileFeed.tsx:104
msgid "Could not load feed"
msgstr "Błąd podczas ładowania kanału"

#: src/view/screens/ProfileList.tsx:1020
msgid "Could not load list"
msgstr "Błąd podczas ładowania listy"

#: src/components/dms/ConvoMenu.tsx:88
msgid "Could not mute chat"
msgstr "Błąd podczas wyciszania rozmowy"

#: src/view/com/composer/videos/VideoPreview.web.tsx:56
msgid "Could not process your video"
msgstr "Błąd podczas przetwarzania wideo"
```